FirebaseMLModelDownloader/Sources/TelemetryLogger.swift

```diff
@@ -126,16 +141,53 @@ class TelemetryLogger {
     fllTransport.sendTelemetryEvent(eventForTransport)
   }
 
+  /// Log model deleted event to Firelog.
+  func logModelDeletedEvent(eventName: EventName, isSuccessful: Bool) {
+    guard app.isDataCollectionDefaultEnabled else { return }
```

Comment:
@ryanwilson can you confirm that this is the right way to respect the Firebase-wide data collection flag in the SDK?